openapi.yml

```diff
@@ -2292,7 +2334,7 @@ components:
           nullable: true
         source:
           oneOf:
-          - $ref: '#/components/schemas/SourceEnum'
+          - $ref: '#/components/schemas/FlawSourceEnum'
```

Comment:
some of these changes don't make sense (e.g. this PR doesn't even touch the `Flaw.source` field), are you using the version of `drf-spectacular` specified in the `requirements.txt` and did you regenerate the schema with the provided script / pre-commit hook?

Reply:
I'm really not sure what happened there either. I didn't edit the schema by hand, it was generated by the pre-commit hook with the version of drf-spectacular installed by `make dev-env`. 

```
(venv) [kgrant@localhost osidb]$ pip list installed | grep drf-spectacular
drf-spectacular                   0.21.0

(venv) [kgrant@localhost osidb]$ python manage.py spectacular --settings=config.settings_local --version
3.2.15
```

If either of those is incorrect, I'll try to figure out why.

Reply:
It's weird but not something worth spending too much time on, I think functionally it's the same